README.md

```diff
@@ -1,37 +1,45 @@
 # Veres One DIDs
 
 This library provides support classes for creating and processing
-Decentralized Identifiers for Veres One. This library enables a developer to:
+Decentralized Identifiers for [Veres One](https://veres.one/). This library
+enables a developer to:
 
 * Create a Veres One DID
-* Perform Veres One key rotation
 * Generate Veres One cryptographic proofs
 
-## The Promises API
-
-  * api.generate(options)
-
-## The Callback API (node.js)
-
-  * api.generate(options, (err, didDocument))
-
-## Quick Examples
-
 ```
 npm install did-veres-one
 ```
 
 ```js
-const didv1 = require('did-veres-one');
-const options = {};
+const v1 = require('did-veres-one');
 
-// generate the DID document
-const didDocument = await didv1.generate();
+// See Configuration below for list of options
+const options = {mode: 'dev', httpsAgent, hostname: 'localhost:12345'};
+const veresDriver = v1.driver(options);
 ```
 
 ## Configuration
 
-For documentation on configuration, see [config.js](./lib/config.js).
+* options - a set of options used when generating the DID Document
+  * didType - the type of DID to generate.
+      Options: 'nym' (default) or 'uuid'
+  * `invokeKey` - optionally pass in a Capability Invocation key, otherwise
+    it will be generated.
+  * keyType - the type of keys to generate.
+      Options: 'Ed25519VerificationKey2018' (default) or
+      'RsaVerificationKey2018'
```

Comment:
I recommend we remove `RsaVerificationKey2018` from the generation options documentation. It's ok to keep the support in there as hidden for now, but I suspect we may not allow RSA based `did:v1:nym` DIDs. That doesn't preclude one from adding the keys after the DID is generated, but RSA-based nyms are messier for a lot of reasons and don't come with the same feature set. Also, ed25519 is slated to get NIST approval in the next fips publication -- should that matter.

Reply:
Removed.